packages/gui/src/components/offers2/OfferIncomingTable.tsx

```diff
@@ -211,17 +199,7 @@ export default function OfferIncomingTable(props: OfferIncomingTableProps) {
 
   async function handleDeleteNotification(notification: Notification) {
     try {
-      const canProcess = await openDialog(
```

Comment:
The `deleteNotification` code path displays its own confirmation dialog, so this confirmation is redundant.